components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/jms/JMSPollingConsumer.java

```diff
@@ -265,9 +265,11 @@ public Message poll() {
             }
 
         } catch (JMSException e) {
-            logger.error("Error while receiving JMS message. " + e.getMessage(), e);
-        } catch (Exception e) {
-            logger.error("Error while receiving JMS message. " + e.getMessage(), e);
+            logger.error("Error while receiving JMS message for " + name, e);
+            releaseResources(true);
+        }
+        catch (Exception e) {
```

Comment:
Formatting issue. '}' and 'catch' should be in the same line

Reply:
Thanks. Fixed.